projects/contributor-experience.md

```diff
@@ -0,0 +1,115 @@
+aspects as individuals and not in a team effort, notable:
+
+* [Add skeleton for maintainer and contributor guidebooks](https://github.com/open-telemetry/community/pull/2051)
+* Mentorship efforts, e.g. [Outreachy](https://cloud-native.slack.com/archives/C060GFUL0P6)
+* Surveys by the End User SIG that aim towards contributor experience
+* _add more_
+
+Also related to this initiative is the [Check-In process of the Governance Committee](https://github.com/open-telemetry/community/blob/main/gc-check-ins.md)
+
+### Goals, objectives, and requirements
+
+The goal of this project is to establish a cross-cutting "SIG Contributor Experience",
+which is "responsible for improving the experience of those who upstream contribute to
+the OpenTelemetry project. [They] do this by creating, and maintaining programs and processes that promote community health and reduce project friction, while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed". (via [K8s Contributor Experience Special Interest Group Charter](https://github.com/kubernetes/community/blob/master/sig-contributor-experience/charter.md))
+
+While this is "yet another SIG" initiative, we believe that **not** establishing
+such a SIG is not an option: the problems outlined above will get bigger not
+smaller and while the final ownership of the health of the project is with the GC
+this requires a coordinated effort and more hands across the project to be addressed
+successfully.
+
+Note that similar to other cross-cutting SIGs (Docs, Security, End-User) this SIG will not exclusively own
+the domain of contributor experience, but they take responsibility to drive efforts in close collaboration
+with SIGs and Committees.
+
+## Deliverables
+
+After foundation the SIG will collaborate with contributors of existing efforts
+(see above) and work towards establishing programs and initiatives around contributor
+experience, including
+
+* Onboarding rituals for members, approvers, maintainers
+* A contributor guidebook
+* Mentorship opportunities
+* Contributor recognition programs (e.g. via [Credly](https://credly.com) or with blog posts on the style of "here's what's happening under the hood"
+* Blog posts and social media posts to highlight areas where help is needed
+* Events targeting the Contributor Community
+* Consume SIG Check-In reports from the GC and use this data to improve contributor experience
+
+## Staffing / Help Wanted
+
+GC/TC sponsors:
+
+* [@jpkrohling](https://github.com/jpkrohling)
+* [@svrnm](https://github.com/svrnm)
+  
+Maintainers, approvers, and contributors:
+
+* [@svrnm](https://github.com/svrnm)
+* [@jpkrohling](https://github.com/jpkrohling)
+* [@mx-psi](https://github.com/mx-psi)
+* [@theletterf](https://github.com/theletterf)
+* [@JamieDanielson](https://github.com/JamieDanielson)
```

Comment:
```suggestion
* [@musingvirtual](https://github.com/musingvirtual)
```

Reply:
@svrnm Looks like I was accidentally removed from this 😅

Reply:
oh no, I am sorry! Let me fix this